tests/Doctrine/Tests/ORM/Functional/Ticket/DDC2090Test.php

```diff
@@ -47,13 +49,11 @@ public function testIssue(): void
             ->set('e.startDate', ':date')
             ->set('e.salary', ':salary')
             ->where('e = :e')
-            ->setParameters(
-                [
-                    'e'      => $employee1,
-                    'date'   => $date1,
-                    'salary' => 101,
-                ]
-            )
+            ->setParameters(new ArrayCollection([
+                new Parameter('e', $employee1),
+                new Parameter('date', $date1),
+                new Parameter('salary', 101),
+            ]))
```

Comment:
This test used the deprecated API of `setParameters()` which I'd like to remove on 3.0.